integers:
  integer3: 3
doubles:
  double34: 3.4
strings:
  stringabc: "abc"
